### DISCUSSION TITLE
Offline files for phone browser (chrome, Firefox, etc)

### DISCUSSION THREAD

**Discussion by PhongTong** [SethClydesdale/genki-study-resources#157] [Questions]
I can use your offline files just fine in PC browser. However it can't work when using phone browser. I've already tried using different browser: Chrome, Firefox, Samsung Internet, Opera. But no luck

**Comment by SethClydesdale** [comment]
Hello @PhongTong,

Unfortunately I do not own a phone, so I cannot really test or provide an adequate solution.. but I'm assuming the local file system is not accessible via the browser? On some systems, local files can usually be accessed by specifying the local directory in the address bar, `file:/genki-study-resources/index.html` for example, though it may vary or not even be possible depending on the system..

**Comment by PhongTong** [comment]
I open index.html file directly in file management app

https://imgur.com/a/JwaEf5X

After that, I choose one of many browsers to open it, this time I choose Chrome app

https://imgur.com/HVwrHiA

The html file can be open normally

https://imgur.com/yRlPfPP

However when I choose a lesson, the browser send a error note: ERR_FILE_NOT_Found

https://imgur.com/9lxhTzL

The address bar of lesson shows: 

content://0@media/external/file/lessons/lesson-1/vocab-1/

**Comment by SethClydesdale** [comment]
Ah, I see... From the looks of it, the necessary resources aren't being loaded: CSS (for design) and JS (for functionality). I'll take a look tomorrow and see if there's any similar issues out there that I can reference and perhaps find a solution.

**Reply by PhongTong** [reply]
Sorry but any news

**Reply by SethClydesdale** [reply]
Unfortunately I couldn't find a solution. I apologize m(__)m

**Reply by yoiang** [reply]
If I understand the situation @PhongTong is working within, rather than being able to access a file path on the device I think  most mobile applications share in-memory pieces of content (image, text file, etc) directly between applications for a number of reasons, including simplicity and security, and this share api usually is only one piece of content at a time. I think that means this might be possible if there was a build process that embedded the whole application, script, images, everything, in one `html`
